### DISCUSSION TITLE
Dataset format for AI (A 4k Star Repo)

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#279] [Ideas]
Hi Haiping,

I've stumbled upon your GitHub profile and noticed you are interested in awesome computer vision projects. :) My team and I are working on the open-source package [Hub](http://github.com/activeloopai/Hub), a dataset format specifically for deep learning + an API for creating, storing, and collaborating on AI datasets of any size. With Hub, one can store petabyte-scale datasets as single numpy-like arrays on the cloud. Thanks to the format, you can stream datasets while training in PyTorch, or get started with datasets as big as ImageNet in less than 5 seconds.

Can you please take a look at it and say what do you think of the project? We want the package to be as useful as possible to the community.

Thanks a lot,
Mikayel

**Comment by RaivoKoot** [comment]
I'm already in love with activeloop Hub. The first 15 minutes of this video give an overview (https://www.youtube.com/watch?v=x8pi51ZBb8o&ab_channel=Bugoutdev%E2%80%93Powertothedevelopers%21)

It makes working with datasets a lot more organized and robust. Any data-centric project either involving multiple team members or large-scale data like videos would probably benefit from it. recommend checking it out.
